cv_basics/Hand[] hands
